lib/ndr_import/file/base.rb

```diff
@@ -8,10 +8,11 @@ module NdrImport
   module File
     # All common base file handler logic is defined here.
     class Base
-      def initialize(filename, format, options = {})
-        @filename = filename
-        @format = format
-        @options = options
+      def initialize(filename, format, delimiter, options = {})
```

Comment:
Can we pass the `delimiter` inside `options` instead of as another argument? It would avoid changing the signature of a method that is inherited by many classes for which the option does not make sense.